docs/release-1.2/existing-rhdh.md

```diff
@@ -270,7 +270,7 @@ To import the Orchestrator software templates into the catalog via the Backstage
 Register new templates into the catalog from the
 - [Workflow resources (group and system)](https://github.com/parodos-dev/workflow-software-templates/blob/v1.2.x/entities/workflow-resources.yaml) (optional)
 - [Basic template](https://github.com/parodos-dev/workflow-software-templates/blob/v1.2.x/scaffolder-templates/basic-workflow/template.yaml)
-- [Complex template - workflow with custom Java code](https://github.com/parodos-dev/workflow-software-templates/blob/v1.2.x/scaffolder-templates/complex-assessment-workflow/template.yaml)
+- [Advanced template - workflow with custom Java code](https://github.com/parodos-dev/workflow-software-templates/blob/v1.2.x/scaffolder-templates/advanced-workflow/template.yaml)
```

Comment:
do we really want to backport the naming change to 1.2?

Reply:
Good point. Are the software templates going to be backported? I just wanted to avoid broken links, in the case we erase all instances of 'complex-assesment'

Reply:
IMO we should not backport it, @masayag WDYT?

Reply:
Agree, this should not be backported.

Reply:
no backport to 1.2, and not to 1.3. 1.3 is nearly out and we should change it for non-critical fixes or enhancements.
IIRC, this entire effort (software templates improvement) is targeted to milestone-4, therefore it should go to main and will be released as part of 1.4 release.